modules/game_actionbar/game_actionbar.lua

```diff
@@ -972,7 +972,7 @@ function onSpellGroupCooldown(groupId, duration)
             end
         end
         if spell then
-            if table.contains(spell.group, groupId) then
+            if spell.group[groupId] ~= nil then
```

Comment:
```suggestion
            if spell.group[groupId] then
```

Reply:
This is cleaner and more idiomatic in Lua than explicitly comparing with nil.

Reply:
Agree with this being more idiomatic. But won't this return false when the spell has a group cooldown of 0s?

I admit it's a bit of an edge case, but it is technically a valid config value (the spell belongs to a spell group and cannot be cast while that group is on cooldown, but casting that spell does not actually cause that group to go on cooldown).

Reply:
@glecko

From what I saw, group is an object, so it can only be a [table] or nil.
https://github.com/mehah/otclient/blob/a189d58fa1ac61533c8680d7967efd32fcfbceac/modules/gamelib/spells.lua#L65..

Reply:
Yes, but `spell.group[groupId]` can either be nil or a number.

Reply:
> Yes, but `spell.group[groupId]` can either be nil or a number.

correct, sry, my mistake.